tests/client/amo/components/TestFeaturedAddons.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+
+import * as featuredActions from 'amo/actions/featured';
+import {
+  FeaturedAddonsBase,
+  mapStateToProps,
+} from 'amo/components/FeaturedAddons';
+import createStore from 'amo/store';
+import { ADDON_TYPE_EXTENSION, ADDON_TYPE_THEME } from 'core/constants';
+import { fakeAddon } from 'tests/client/amo/helpers';
+import { getFakeI18nInst, shallowRender } from 'tests/client/helpers';
+
+
+describe('<FeaturedAddons />', () => {
+  const initialState = { api: { clientApp: 'android', lang: 'en-GB' } };
```

Comment:
This should use [signedInApiState](https://github.com/mozilla/addons-frontend/blob/70f359d87cb70e2ecf2edf905ffedbb3e661e778/tests/client/amo/helpers.js#L45) like `{ api: signedInApiState }`